src/jit/SimdArm32Inl.h

```diff
@@ -63,6 +66,7 @@ enum Type : uint32_t {
     vqsub = 0xef000250,
     vshlImm = 0xef800550,
     vshl = 0xef000440,
+    vsqrt = 0xeeb108c0,
```

Comment:
This is just a few instructions, could we add the arm32 variant for them?

Reply:
Btw vsqrt is approximate square root. Is it good enough for WebAssembly? It expects precise results in other cases.